dashboard/db/migrate/20210119225102_copy_activity_section_title_to_progression_name.rb

```diff
@@ -0,0 +1,22 @@
+class CopyActivitySectionTitleToProgressionName < ActiveRecord::Migration[5.2]
+  def up
+    Script.where("properties -> '$.is_migrated' = ?", true).each do |script|
+      script.lessons.each do |lesson|
+        lesson.lesson_activities.each do |activity|
+          activity.activity_sections.each do |section|
+            next if section.script_levels.empty?
+            section.progression_name = section.name
+            section.save!
+            section.script_levels.each do |sl|
+              sl.progression = section.name
+              sl.save!
+            end
+          end
+        end
+      end
+    end
```

Comment:
another idea would be to take all the code inside this migration and move it to a oneoff script, and then add a line here which rewrites the .script_json before moving onto the next script. then you could just run this on your local machine (or levelbuilder) to get the change you want into the .script_json files.